frontend/src/store/facilityBooking/action.ts

```diff
@@ -328,6 +328,7 @@ export const handleCreateBooking = (isEdit: boolean) => async (dispatch: Dispatc
   } else {
     const { newBooking } = getState().facilityBooking
     const TokenId = localStorage.getItem('token')
+    requestBody.facilityID = newBooking?.facilityID ? newBooking?.facilityID : requestBody.facilityID
```

Comment:
consider `requestBody.facilityID = newBooking?.facilityID ?? requestBody.facilityID` instead?

Reply:
Fixed in 7889fdc